controllers/usersignup/masteruserrecord.go

```diff
@@ -1,13 +1,14 @@
 package usersignup
 
 import (
+	"strings"
+
 	toolchainv1alpha1 "github.com/codeready-toolchain/api/api/v1alpha1"
-	tierutil "github.com/codeready-toolchain/host-operator/controllers/nstemplatetier/util"
 
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-func migrateOrFixMurIfNecessary(mur *toolchainv1alpha1.MasterUserRecord, nstemplateTier *toolchainv1alpha1.NSTemplateTier, userSignup *toolchainv1alpha1.UserSignup) (bool, error) {
+func migrateOrFixMurIfNecessary(mur *toolchainv1alpha1.MasterUserRecord, defaultTier *toolchainv1alpha1.NSTemplateTier, userSignup *toolchainv1alpha1.UserSignup) bool {
```

Comment:
Wouldn't it be safer to get the actual resource to make sure `config.Tiers().DefaultTier()` is set correctly?

Reply:
I guess that it would just result in an error in the MUR status which would be expected behaviour so I'll go ahead and switch to using the name only.

Reply:
> Wouldn't it be safer to get the actual resource to make sure config.Tiers().DefaultTier() is set correctly?

That's a good point - we shouldn't maybe set the tier if it doesn't exist (in case it is set incorrectly). I'll let you do whatever you think that is better/safer :wink:

Reply:
technically, if you wanted to check it before setting it in a `Space` to be 100% that the tier really exists, then you could do only at one place inside of the `ensureSpace` method. But it's up to you.

Reply:
I kept this the same, checking inside ensureSpace was too many more changes 😂  Can do it later if we really want it